submissions/phpslonuk/JS_DOM/script/main.js

```diff
@@ -0,0 +1,141 @@
+	},
+	{
+		image: "https://raw.githubusercontent.com/phpslonuk/JS_DOM/main/img/Japan.jpg",
+		title: "Різдво в Японії",
+		alt: "Japan",
+		description: "В Японії замість Діда Мороза основна фігура свята – бог Хотейошо, у якого очі… на потилиці. Традиційне різдвяне меню японців – рис, боби, гриби і бамбук. І, звичайно, улюблений веселий напій саке. Ще на Різдво японці біля своїх будинків розвішують на високих жердинах пучки соломи і підпалюють їх, щоб відлякати злих духів."
+	},
+	{
+		image: "https://raw.githubusercontent.com/phpslonuk/JS_DOM/main/img/USA.jpg",
+		title: "Різдво в США",
+		alt: "USA",
+		description: "Сполучені штати – багатонаціональна країна, тому можна побачити різні способи святкування Різдва. Деякі готують традиційну фаршировану індичку. Інші вважають, що куди  краще ковбаски, страви з капусти та супи. Італійці готують на Різдво лазанью. Діти отримують подарунки вранці. Їм кажуть, що вночі приходив Санта-Клаус, він спустився димоходом і приніс подарунки. У різдвяний ранок традиційно подають рулет з корицею і каву, а на обід – шинку. Американці – нація, яка чи не найбільше отримує і дарує подарунків у це свято. У невеликих містах Америки прийнято ходити один до одного в гості та всім разом співати різдвяні пісні. У США будь-які заклади,  чи то банк, магазин чи фірма, зачиняються. Всі відпочивають і найчастіше дивляться по телевізору «Лускунчика» і «Червону шапочку»."
+	}
+]
+
+
+const doWithElement = function (db, index) {
+	document.querySelector(".title").innerHTML = db[index].title;
+	document.querySelector(".country-img").src = db[index].image;
+	document.querySelector(".country-description").innerHTML = db[index].description;
+
+
+}
+
+const getElementAttribute = function (e) {
+	let index = e.target.dataset.index;
+	doWithElement(DATABASE, index);
+
+}
+
+const createContent = function () {
+	let content = document.querySelector(".main");
+	let article = document.createElement("article");
+	article.className = "article";
+	article.innerHTML = `
+	    <div>
+        <h2 class="title">Kottans i wish you a Merry Christmas</h2>
+        </div>
+        <p class="country-description">Привіт всім, я радий вас тут вітати, бажаю всім пройти цей курс і
+          отримати від нього максимум знань і задоволення. Можливо не всім вийде попасти на наступний рівень
+          але Новий Рік і Різдво ніхто не пропустить, тому всього вам найкращого у наступному році КОТАНИ
+          !!!!!</p>
+        <img class="country-img" src="https://raw.githubusercontent.com/phpslonuk/JS_DOM/main/img/kitten-sleeping.jpg" alt="country">`;
+
+	let navigation = document.createElement("nav");
+	navigation.className = "navbar";
+	let navList = document.createElement("ul");
+	navList.className = "main-menu";
+
+	DATABASE.forEach(function (item, i) {
+		let li = document.createElement("li");
+		li.className = "nav-list";
+		li.setAttribute("data-index", i);
+		li.addEventListener("click", getElementAttribute)
```

Comment:
Use [event delegation](https://learn.javascript.ru/event-delegation) for listener